serverless.js

```diff
@@ -149,6 +169,14 @@ module.exports = {
           },
         },
       ],
+      environment: {
+        AUTH0_CLIENT_ID: `\${ssm:auth0-client-id-${
```

Comment:
was your intention to hide these? basically right now these two are exposed as follows:
![image](https://user-images.githubusercontent.com/11645956/114725978-ddcd1f80-9d34-11eb-99d3-986351b113b4.png)
if you dont think these values are secret then they can stay in SSM and be injected into the environment through the environment variable, just like you did it.